src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Tizen.Network.Stc
+{
+    /// <summary>
+    /// A class for managing the Stc Info (statistics information).
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+
+    public class NetworkStatistics : IDisposable
+    {
+        private IntPtr _infoHandle = IntPtr.Zero;
+        private bool _disposed;
+
+        internal NetworkStatistics(IntPtr handle)
+        {
+            Log.Debug(Globals.LogTag, "New Info. Handle: " + handle);
+            _infoHandle = handle;
+        }
+
+        /// <summary>
+        /// Destroy the NetworkStatistics object
+        /// </summary>
+        ~NetworkStatistics()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// A method to destroy the managed NetworkStatistics objects.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        protected void Dispose(bool disposing)
+        {
+            if (_disposed)
+            {
+                return;
+            }
+
+            if(disposing)
+            {
+                // destroy managed resources
+            }
+
+            int ret = Interop.Stc.Info.StatsDestroy(_infoHandle);
+            if (ret == (int)StcError.None)
+            {
+                _infoHandle = IntPtr.Zero;
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Failed to destroy network statistics handle, Error - " + (StcError)ret);
+            }
+
+            _disposed = true;
+        }
+
+        /// <summary>
+        /// A property to get the application ID from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Application ID.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
```

Comment:
Wrong comment. this property only provides getter. So when try set any value to this property. It makes a complication error.

Reply:
Corrected description of such properties. Thank you.